stan-dev/stan
# Bugfix/issue 1223 pareto type 2 log bad partials

#### Summary:

Fixes the improper calculation of partial derivatives of the pareto_type_2_log with respect to the first three parameters. Addresses #1223 

#### Intended Effect:

Finite diff tests for pareto_type_2_log should no longer fail at non-zero entries for the location parameter, mu.

#### How to Verify:

Verify that the valid_value method for AgradDistributionParetoType2 in src/test/unit-distribution/univariate/continuous/pareto_type_2/pareto_type_2_test.hpp has a non-zero entry in parameters[0][1] which corresponds to mu, and run pareto_type_2_00000_generated_v_test.cpp. This test should run without any failures. 

#### Side Effects:

None.

#### Documentation:

None required.

#### Reviewer Suggestions: 

Anyone.
